### ISSUE TITLE
Specific configuration of TinyMCE is not working on every Text Editor

### ISSUE THREAD

**Issue by PrestaEdit** [PrestaShop/PrestaShop#36351]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

It's seems that ProductPage V2 and also CMS page could not take this benefits (https://devdocs.prestashop-project.org/9/development/components/tinymce/)

Was working great on Product Page v1 (v8.1).

### Expected behavior

Having the possibility to use the config extended on every page.

### Steps to reproduce

1. Add a module as mention on the devdocs
2. Go to product page
3. See nothing changed

### PrestaShop version(s) where the bug happened

9.0.0-alpha.1

### PHP version(s) where the bug happened

8.2

**Comment by florine2623** [comment]
ping @kpodemski 😄

**Comment by matks** [comment]
Hello @PrestaEdit 

# About your bug report

We are not in your brain. You need to explain better ;
```
Add a module as mention on the devdocs
Go to product page
See nothing changed
```

3 sentences is not enough at all to understand the problem. It might be crystal clear for you because you just observed the problem but from anybody else it's very confused.

If you wanted to really help this bug being reported, THIS is how to write the "how to reproduce":

### Steps to reproduce

1. Create a module. In the module use the hook `actionAdminControllerSetMedia` to include an extra JS / CSS link inside all prestashop pages, as explained on the documentation https://devdocs.prestashop-project.org/9/development/components/tinymce/ . Example 

```
    public function hookActionAdminControllerSetMedia()
    {
        $this->context->controller->addJs('myfile.js');
    }
```

2. Inside `myfile.js` add the following JavaScript code to override the default configuration of the TinyMCE component. For example the below configuration will add the ability to paste text to TinyMCE:
See nothing changed
```
var config = {
    plugins: 'align colorpicker link image filemanager table media placeholder advlist code table autoresize paste',
    toolbar1:
      'code,colorpicker,bold,italic,underline,strikethrough,blockquote,link,align,bullist,numlist,table,image,media,formatselect,pastetext',
  };

window.defaultTinyMceConfig = config;
```

3. Verify on a backoffice page that the "Paste as text" option is now enabled inside a tinyMCE field
![Capture d’écran 2024-07-01 à 22 52 02](https://github.com/PrestaShop/PrestaShop/assets/3830050/f4f9236b-5e6f-48fa-bdd5-61916a61fcf4)

4. Now go to backoffice "Edit a product" page
5. See that the tinyMCE fields do NOT benefit from the "Paste as text" option on the backoffice "Edit a product page"

# About the issue

Now that I have explained how to submit this bug report, I have tested this and on my shop this works perfectly. Here you can see my backoffice "Edit a product" page with "Paste as text" TinyMCE option enabled, thanks to the above module.


![Capture d’écran 2024-07-01 à 22 53 15](https://github.com/PrestaShop/PrestaShop/assets/3830050/cad2b40d-48c6-422c-9148-ad46d237e4c6)

So on my shop Product Page V2 works very well with the TinyMCE config extension system.

**Comment by PrestaEdit** [comment]
Hi @matks,

Maybe cause the issue was solved by this : https://github.com/PrestaShop/PrestaShop/pull/36433

I will have a look on this when I've more time.

**Comment by paulnoelcholot** [comment]
Hello @PrestaEdit,

Since we had no news from you for more than 20 days, I'll close this ticket to avoid cluttering up the backlog. Please note that you can always create a new one if further information pops up.

Thank you

**Comment by kreare** [comment]
Any chance to get this fixed on v8.1 branch? Seems to be merged only on develop branch (thus, for v9?)
